src/indexer/segment_updater.rs

```diff
@@ -91,6 +91,7 @@ fn merge(
     index: &Index,
     mut segment_entries: Vec<SegmentEntry>,
     target_opstamp: Opstamp,
+    segment_attributes: Option<serde_json::Value>,
```

Comment:
I don't understand what this is thing is. Is it to override the merge segment attributes?

Reply:
Fixed